app/Http/Controllers/VisitTransfer/Admin/VisitorStatsController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers\VisitTransfer\Admin;
+
+use App\Models\Mship\Account as Accounts;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\Input;
+
+class VisitorStatsController extends \App\Http\Controllers\Adm\AdmController
+{
+    /**
+     * @var Accounts
+     */
+    protected $accounts;
+
+    public function __construct(Accounts $accounts)
+    {
+        parent::__construct();
+
+        $this->accounts = $accounts;
+    }
+
+    public function index()
+    {
+        $inputStartDate = Input::get('startDate');
+        $inputEndDate = Input::get('endDate');
```

Comment:
Can we take advantage of Carbon/Request here so that a date can be inputted but it converts it correctly for the query?
Like the below?

https://github.com/VATSIM-UK/core/blob/93c0085512563fbda82fb1139553c383c4280fd7/app/Http/Controllers/Adm/Mship/Feedback.php#L267-L274

Reply:
We could if it was a post request - the Date submits via a get request. I'm happy to do a new carbon object rather than parsing it though if that is what you're getting at?

Reply:
I'd rather we just posted, there isn't any real benefit in doing it via GET as I doubt people will bookmark the URL with the query string.

Perhaps we could simplify this a little then by forcing the user to enter a date before an output is displayed? That way, we can just grab the request, perform the query and return rather than defaulting the `$from_date` and `$to_date`